pkgs/data/icons/comixcursors/default.nix

```
@@ -0,0 +1,135 @@
+  #
+  # When not all of the variants, i.e. only a smaller subset of them, are
+  # desired (i.e. when a subset of outputs are chosen), install-all will still
+  # build all of them.  While upstream appears to provide old functionality for
+  # building only a subset, it is broken and we do not use it.  With prebuilt
+  # substitutions, installers of this package will get only the outputs they
+  # chose.
+  buildPhase = ''
+    ICONSDIR=$TMP/staged ./install-all
+  '';
+
+  installPhase = ''
+    for outputName in $outputs ; do
+        if [ $outputName != out ]; then
+            local outputDir=''${!outputName};
+            local iconsDir=$outputDir/share/icons
+            local cursorName=$(tr _ - <<<"$outputName")
+
+            mkdir -p $iconsDir
+            cp -r -d $TMP/staged/ComixCursors-$cursorName $iconsDir
+
+            unset outputDir iconsDir cursorName
+        fi
+    done
+
+    # Need this directory (empty) to prevent the builder scripts from breaking.
+    mkdir -p $out
+  '';
+
+  outputs = let
+    default = "Opaque_Black";
+  in
+    # Have the most-traditional variant be the default output (as the first).
+    # Even with outputsToInstall=[], the default/first still has an effect on
+    # some Nix tools (e.g. nix-build).
+    [ default ] ++ (remove default variants)
+    # Need a dummy "out" output to prevent the builder scripts from breaking.
+    ++ [ "out" ];
+
+  # No default output (to the extent possible).  Instead, the outputs'
+  # attributes are used to choose which variant(s) to have.
+  outputsToInstall = [];
+
+  meta = with lib; {
+    description = "The original Comix Cursors mouse themes";
+    longDescription = ''
+      There are many (${toString ((length outputs) - 1)}) variants of color,
+      opacity, edge thickness, and right- or left-handedness, for this cursor
+      theme.  This package's derivation has an output for each of these
+      variants, named following the upstream convention, and the attribute for
+      an output must be used to install a variant.  E.g.:
+      <programlisting language="nix">
+      environment.systemPackages = [
+        ${pname}.Blue
+        ${pname}.Opaque_Orange
+        ${pname}.Slim_Red
+        ${pname}.Opaque_Slim_White
+        ${pname}.LH_Green
+        ${pname}.LH_Opaque_Black
+        ${pname}.LH_Slim_Orange
+        ${pname}.LH_Opaque_Slim_Blue
+      ];
+      </programlisting>
+
+      Attempting to use just <literal>${pname}</literal>, i.e. without an output
+      attribute, will not install any variants.  To install all the variants,
+      use <literal>${pname}.all</literal> (which is a list), e.g.:
+      <programlisting language="nix">
+      environment.systemPackages = ${pname}.all ++ [...];
+      </programlisting>
+
+      The complete list of output attributes is:
+      <programlisting>
+      ${concatStringsSep "\n" variants}
+      </programlisting>
+    '';
+    homepage = "https://gitlab.com/${owner}/${repo}";
```

```suggestion
    homepage = "https://gitlab.com/limitland/comixcursors";
```
otherwise I can't click on it.